students/AndyKwok/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,16 @@
+#Description: Program that prints out Fizz upon division of 3 and Buzz upon division of 5
```

Comment:
it isn't a big deal now since these are scripts.. but python modules (aka python files) should follow snake_case convention -> `fizz_buzz.py`